src/vdom/builder.rs

```diff
@@ -38,31 +38,64 @@ impl MountPoint for web_sys::HtmlElement {
 }
 
 /// Used for handling initial routing.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
 pub enum UrlHandling {
     PassToRoutes,
     None,
     // todo: Expand later, as-required
 }
 
+/// Describes the handling of elements already present in the mount element.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum MountType {
```

Comment:
Add `MountType` into prelude please:
Replace:
https://github.com/David-OConnor/seed/blob/179e11d3238ab68eb265dc38646099da01ad3631/src/lib.rs#L100
With:
```rust
vdom::{Init, UrlHandling, MountType},
```

Reply:
Got it.